### PR TITLE
Dropdown Component

### PR DESCRIPTION
**Pull request by harshagarwal08**
1. Dropdown Button
Screenshot 2023-03-04 at 3 41 10 PM
2. On clicking the button, it will show a list of options which will turn the background black on hover
Screenshot 2023-03-04 at 3 42 05 PM
3. Search functionality for dropdown options
Screenshot 2023-03-04 at 3 42 27 PM
4. Shows selected option as Dropdown name. This will also send the selected option to the parent component which can then be used to filter data based on the selected option.
Screenshot 2023-03-04 at 3 42 41 PM

### CODE REVIEW

**Review by rohini-senthil — APPROVED** [review]

File: src/components/Dropdown/index.jsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+            placeholder="search"
+            value={inputValue}
+            onChange={e => setInputValue(e.target.value.toLowerCase())}
+            className="placeholder:text-gray-700 outline-none text-mg w-36 pr-2"
+          />
+        </div>
+        {dropdownData.map(option => (
+          <li
+            key={option}
+            className={`p-2 text-sm hover:bg-black hover:text-white cursor-pointer
```
Comment (rohini-senthil) [inline src/components/Dropdown/index.jsx:33]: Instead of the hover being black can we have some kind of grey instead?

Comment (harshagarwal08) [reply src/components/Dropdown/index.jsx:33]: yeah sure. I thought it would go with the overall page design

Comment (harshagarwal08) [reply src/components/Dropdown/index.jsx:33]: Screenshot 2023-03-04 at 6 45 46 PM
which one would look good?

Comment (harshagarwal08) [reply src/components/Dropdown/index.jsx]: gave gray-300 background color for now

**Review by rohini-senthil — APPROVED** [review]

**Comment by Chetan-MckinseyBEBR** [comment]
image
@harshagarwal08 can u add a border to the dropdown list?

**Comment by Chetan-MckinseyBEBR** [comment]
image

@harshagarwal08 dropdown shouldn't take that height. The height of the drop-down should be equivalent to the select input field. If you add this in the nav bar, it will overflow

**Pull request merged**